### ISSUE TITLE
Uncaught exception, program will not run

### ISSUE THREAD

**Issue by fuzzysh** [jeffvli/sonixd#255]
**Describe the bug**
Sonixd will not launch

**To Reproduce**
I am running the command `./Downloads/Sonixd-0.13.1-linux.AppImage`

**Desktop (please complete the following information):**
 Void 5.15.26_1 x86_64 GenuineIntel uptodate rrmFFFFFFFFFF

**Additional context**
program prints out the following:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: unknown address type:disabled
    at createStream (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/dbus-next/lib/connection.js:64:17)
    at createConnection (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/dbus-next/lib/connection.js:81:33)
    at createClient (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/dbus-next/index.js:10:20)
    at Object.sessionBus (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/dbus-next/index.js:41:10)
    at Player.init (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/mpris-service/src/index.js:183:20)
    at new Player (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/mpris-service/src/index.js:175:8)
    at Player (/tmp/.mount_SonixdWiXoiN/resources/app.asar/node_modules/mpris-service/src/index.js:168:12)
    at Module.78118 (/tmp/.mount_SonixdWiXoiN/resources/app.asar/main.prod.js:2:1258494)
    at n (/tmp/.mount_SonixdWiXoiN/resources/app.asar/main.prod.js:2:1303129)
    at /tmp/.mount_SonixdWiXoiN/resources/app.asar/main.prod.js:2:1303640
```

and continues to run until terminated

**Comment by jeffvli** [comment]
I think same issue as #153, but I thought it was resolved. Can you try the workaround listed there?

**Comment by fuzzysh** [comment]
workaround resolved the issue and the program runs. here's the output

`++ echo
++ awk -F '[=,]' '{ print $2 }'
+ DBUS_PATH=
+ rm -f /tmp/dbus-session-for-sonixd
+ SOCAT_PID=7151
+ trap 'kill -9 $SOCAT_PID' EXIT
+ sleep 0.5
+ socat UNIX-LISTEN:/tmp/dbus-session-for-sonixd,fork ABSTRACT-CONNECT:
+ DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/dbus-session-for-sonixd
+ Sonixd
2022/03/09 16:50:27 socat[8143] E connect(5, AF=1 "\0", 3): Connection refused
A JavaScript error occurred in the main process
Uncaught Exception:
Error: write EPIPE
    at afterWriteDispatched (internal/stream_base_commons.js:156:25)
    at writeGeneric (internal/stream_base_commons.js:147:3)
    at Socket._writeGeneric (net.js:785:11)
    at Socket.connect (net.js:767:12)
    at Object.onceWrapper (events.js:421:28)
    at Socket.emit (events.js:327:22)
    at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1132:10)
2022/03/09 16:50:27 socat[8391] E connect(5, AF=1 "\0", 3): Connection refused
2022/03/09 16:50:27 socat[8410] E connect(5, AF=1 "\0", 3): Connection refused
16:50:27.680 › Checking for update
2022/03/09 16:50:27 socat[8437] E connect(5, AF=1 "\0", 3): Connection refused

(sonixd:7502): LIBDBUSMENU-GLIB-WARNING **: 16:50:27.715: Unable to get session bus: Error receiving data: Connect

(sonixd:7502): libappindicator-WARNING **: 16:50:27.715: Unable to get the session bus: Error receiving data: Conn
16:50:29.469 › Update for version 0.13.1 is not available (latest version: 0.13.1, downgrade is disallowed).
16:50:29.470 › checkForUpdatesAndNotify called, downloadPromise is null
`

thanks for the quick help!
